src/web-auth/hosted-pages.js

```diff
@@ -79,11 +86,23 @@ HostedPages.prototype.login = function(options, cb) {
   );
 
   usernamePassword = new UsernamePassword(this.baseOptions);
+
   return usernamePassword.login(params, function(err, data) {
     if (err) {
       return cb(err);
     }
-    return usernamePassword.callback(data);
+
+    function doAuth() {
+      return usernamePassword.callback(data);
+    }
+
+    if (typeof options.onRedirecting === 'function') {
+      return options.onRedirecting(function() {
+        return doAuth();
+      });
+    }
```

Comment:
I guess this doesnt matter much as the return value of this `login` method isn't used... But our docs do not tell the user to use `return done()`, so `return options.onRedirecting(...)` will always return `undefined` and not the result of `doAuth`.

Reply:
It's a good point, I was focussed on trying not to interrupt the flow of code as much as possible (the code I was changing returned a value) but you're right; we don't use it. Good spot! Let me remove the returns here 👍